### ISSUE TITLE
"Server-side mod" instead of "Server Launcher"

### ISSUE THREAD

**Issue by Neodark7** [copygirl/PackOPdater#9]
Hi :) can you do that ? :D i wanna keep mcmyadmin for launching my server :3

**Comment by copygirl** [comment]
Yeah, that would be the proper solution and I've asked a few people to make such a mod, so far no luck.
